include/git2/submodule.h

```diff
@@ -166,7 +157,7 @@ typedef struct git_submodule_update_options {
 #define GIT_SUBMODULE_UPDATE_OPTIONS_INIT \
 	{ GIT_SUBMODULE_UPDATE_OPTIONS_VERSION, \
 		{ GIT_CHECKOUT_OPTIONS_VERSION, GIT_CHECKOUT_SAFE }, \
-	GIT_FETCH_OPTIONS_INIT, GIT_CHECKOUT_SAFE, 1 }
+	GIT_FETCH_OPTIONS_INIT, GIT_CHECKOUT_SAFE }
```

Comment:
This doesn't seem quite right?  If I'm reading this right, it looks like you've dropped the last initializer (`allow_fetch`) instead of the second to last (`clone_checkout_strategy`).  Instead shouldn't you remove `GIT_CHECKOUT_SAFE` and leave the `1`?